apps/user_dashboard/src/strores/configs/config.ui.ts

```diff
@@ -0,0 +1,107 @@
+// Seperate shared config from UI config to prevent code mixup between UI and background process code.
```

Comment:
Please rename this file. snake case required unless there is an absolute necessity